src/examples/src/widgets/card/ActionButtonsAndIcons.tsx

```diff
@@ -0,0 +1,29 @@
+import { create, tsx } from '@dojo/framework/core/vdom';
+import Card from '@dojo/widgets/card';
+import Button from '../../../../button/index';
```

Comment:
Same comment for these, should be using `@dojo/widgets/` and no need to specify index (and throughout the examples)